content/tugboat-cli/install-the-cli.md

```diff
@@ -57,3 +90,29 @@ Extract the downloaded `tugboat.tar.gz` file to `/usr/local/bin`
 ```sh
 sudo tar -zxf tugboat.tar.gz -C /usr/local/bin/
 ```
+
+### Homebrew
```

Comment:
This is a duplicate of what is just above, and reads as if it is part of the Linux instructions. Can we just remove this section?